Resources/Prototypes/Catalog/Fills/Lockers/suit_storage.yml

```
@@ -319,3 +319,26 @@
         - id: OxygenTankFilled
         - id: ClothingOuterHardsuitWizard
         - id: ClothingMaskBreath
+
+#Cybersun Armoring Corporation
+- type: entity
+  id: SuitStorageShanlin
+  parent: SuitStorageBase
+  suffix: CSA-51a
+  components:
+  - type: StorageFill
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 6 but found 2

(indentation)